tasks/system_probe.py

```diff
@@ -504,6 +521,8 @@ def ninja_generate(
             ninja_security_ebpf_programs(nw, build_dir, debug, kernel_release)
             ninja_container_integrations_ebpf_programs(nw, co_re_build_dir)
             ninja_runtime_compilation_files(nw, gobin)
+            if instrument_trampoline:
```

Comment:
In what scenario do we want this flag `False`? The `build` task is defaulting to true, which is what is used the majority of the time. Just wondering if we can simplify by removing an option. I don't think it hurts to always build this program, right?

Reply:
Agreed. Removed the check.